Templates/ODataUnitTests.tpl

```diff
@@ -306,7 +306,7 @@ namespace <NAMESPACE>
             getResponse.EnsureSuccessStatusCode()
 
             ;;Deserialize the JSON into a <StructureNoplural> object
-            do<StructureNoplural> = JsonConvert.DeserializeObject<<StructureNoplural>>(getResult)
+            do<StructureNoplural> = JsonConvert.DeserializeObject<<IF NOT STRUCTURE_HAS_UNIQUE_KEY>Odata</IF><StructureNoplural>>(getResult)<IF NOT STRUCTURE_HAS_UNIQUE_KEY>.Value[0]</IF>
```

Comment:
if the result is a collection why are we only looking at Value[0]?

Reply:
Modified. See 379ed7f7973349ea3be01c8266651b561b820204

The reason why `Value[0]` is chosen is because all the PKs are the same when doing a GET on the PK number.
Now, there is a test to make sure that all the PKs on the GET are the same number, and then moving forward with the rest of the test.